Syllabus.md

```diff
@@ -0,0 +1,272 @@
+  + [BrighterFatterCorrection.ipynb](ImageProcessing/BrighterFatterCorrection.ipynb)
+
+- Other Resources:
+
+  + [processccd.html](https://pipelines.lsst.io/getting-started/processccd.html)
+  + [data repositories](https://pipelines.lsst.io/getting-started/data-setup.html)
+
+(4) Source Detection
+--------------------
+
+- Topics:
+
+  + Source detection
+  + Footprint Sets (and Heavy Footprints)
+  + Deblending?
+  + Alerts
+  + Moving objects
+  + Topics
+  + The butler
+  + DAF framework
+  + Ways to store and access catalogs from disk
+
+- Notes
+
+  + Seeds for Phase 1 development:  Probably need to go through the step of image co-addition - it is in Image coaddition v15 HTML tutorial, but deblender is in Also v15 HTML tutorial could be a nice match, includes sample extraction of the HST data
+
+- Stack Club Resources:
+
+  + [LowSurfaceBrightness.ipynb](SourceDetection/LowSurfaceBrightness.ipynb)
+  + [lsst_stack_deblender.ipynb](Deblending/lsst_stack_deblender.ipynb)
+  + [scarlet_tutorial.ipynb](Deblending/scarlet_tutorial.ipynb)
+
+- Other Resources:
+
+  + [Deblender.ipynb](https://github.com/RobertLuptonTheGood/notebooks/blob/2eeee8b9fe35077387485e488c965f1ea3d39418/Demos/Deblender.ipynb)
+
+(5) Image Coaddition
+--------------------
+
+- Topics:
+
+  + Remapping
+  + PSF homogenization
+  + Masking
+  + Template generation for DIA
+
+- Stack Club Resources:
+
+  + **None**
+
+- Other Resources
+
+  + Small bit in the HSC re-run script
+
+(6) Sky background estimation
+-----------------------------
+
+- Topics:
+
+  + The sky background problem
+  + How is sky background derived?
+  +  Validating the sky background
+
+- Stack Club Resources: 
+
+  + **None**
+
+- Other Resources:
+
+  + https://github.com/lsst-dm-tutorial/lsst2017/blob/master/tutorial.ipynb
+
+(7) PSF estimation 
+------------------
+
+- Topics:
+
+  + Where does the PSF come from?
+  + How is the PSF estimated?
+  + Diffraction spikes
+  + How do we visualize the psf
+
+- Stack Club Resources:
+
+  + [Image_quality_demo.ipynb](Validation/image_quality_demo.ipynb)
+  + PSF and shears?
+
+- Other Resources:
+
+  + [PSF.ipynb](Demos/PSF.ipynb)
+
+(8) Source/Object measurement
+-----------------------------
+
+- Topics:
+
+  + Photometry
+  + Aperture magnitudes
+  + PSF magnitudes
+  + Model magnitudes
+  + Shapes
+  + Light curves
+
+- Stack Club Resources:
+
+  + **None**
+
+- Other Resources:
+
+  + [Source measurement tutorial](https://pipelines.lsst.io/getting-started/photometry.html)
+  + [Kron.ipynb](https://github.com/RobertLuptonTheGood/notebooks/blob/2eeee8b9fe35077387485e488c965f1ea3d39418/Demos/Kron.ipynb)
+
+(9) Astrometric calibration
+---------------------------
+
+- Topics:
+
+  + Internal astrometry
+  + External astrometry
+
+- Projects:
+
+  + David Shupe? Jointcal?
+
+- Stack Club Resources
+
+  + **None**
+
+- Other Resources:
+
+  + **None**
+
+(10) Photometric calibration
```

Comment:
I wonder whether we should merge sessions 9 and 10 into a single "Calibration" session, especially given the lack of progress on this so far?